go/vt/sqlparser/ast.go

```diff
@@ -4055,10 +4055,27 @@ type AliasedTableExpr struct {
 
 type AsOf struct {
 	Time Expr
+	Start Expr
+	End Expr
```

Comment:
Do we need to add `Start` and `End` to `walkSubtree`? (The only thing I know `walkSubtree` gets used for is finding parameters in a prepared statement; not sure that's valid to use here, but probably couldn't hurt to add it.)

Reply:
Good call